### ISSUE TITLE
Avoiding `resemble-image` in the CSS input

### ISSUE THREAD

**Issue by simevidas** [ben-eb/postcss-resemble-image#6]
Instead of having to use a `resemble-image` function in the CSS input, would it be possible to define on which CSS rulesets this plugin runs by passing a selector string? So, you would pass something like `'header'` to the plugin, which would then find all rulesets with this selector in the CSS input and add the gradient background to all `background: url()` declarations. Harry himself writes that this technique is intended for “a very specific image with a very low rate of change”, so there shouldn’t be more than one or two selectors for which this plugin needs to run.

In case it’s not obvious, `resemble-image` in the input CSS is problematic, because it makes the CSS invalid and dependent on the plugin. I don’t have much experience with PostCSS plugins, but it makes sense to me that they should always try to work automatically without requiring changes to the source CSS (like Autoprefixer).

**Comment by ben-eb** [comment]
Sure, absolutely. I don't see why we can't support both use cases. 😄

What do you think of having a `selectors` array? Something like this:

``` js
{
  selectors: ['header']
}
```

``` css
header, footer {
  background: url(path/to/brand.png); /* fallback generated */
}
```

**Comment by simevidas** [comment]
That sounds great 😃

**Comment by ben-eb** [comment]
Released in `2.1.0`. 😄

**Comment by simevidas** [comment]
Cool! I’ll test it today and report back if there are any issues.
